policy/staged_policy_test.go

```diff
@@ -212,3 +213,55 @@ func TestPoliciesListIsDefault(t *testing.T) {
 		require.Equal(t, input.expected, input.pl.IsDefault())
 	}
 }
+
+func TestPoliciesListJSONMarshaling(t *testing.T) {
+	inputs := []struct {
+		pl       PoliciesList
+		expected string
+	}{
+		{
+			pl:       PoliciesList{},
+			expected: "[]",
+		},
+		{
+			pl:       DefaultPoliciesList,
+			expected: `[{"cutoverNanos":0,"tombstoned":false,"policies":null}]`,
+		},
+		{
+			pl: PoliciesList{
+				NewStagedPolicies(
+					0,
+					false,
+					[]Policy{
+						NewPolicy(NewStoragePolicy(10*time.Second, xtime.Second, 6*time.Hour), DefaultAggregationID),
+						NewPolicy(NewStoragePolicy(time.Minute, xtime.Minute, 24*time.Hour), mustCompress(Lower, Upper)),
```

Comment:
Can you do mustCompress(Count, Mean) to avoid having to change Lower and Upper to Min and Max later?